pkg/runtime/runtime-manifest.go

```diff
@@ -233,8 +231,11 @@ func (m *RuntimeManifest) buildSourceData() (map[string]interface{}, error) {
 		// See https://github.com/deislabs/porter/issues/855
 		m.setSensitiveValue(stepOutput)
 	}
+
 	// Iterate through the bundle-level manifests and resolve for interpolation
-	for _, outputDef := range m.Outputs {
+	for _, outputDef := range m.GetTemplatedOutputs() {
+		// TODO: look into how applyTo will make this throw errors in some cases
```

Comment:
Is this https://github.com/deislabs/porter/issues/1159 ?

Reply:
Yup, will update the comment now that I have an issue created.